docs/topics/api/licenses.rst

```diff
@@ -26,12 +26,16 @@ Open source license slugs are taken from the `SPDX License list <https://spdx.or
     ===================  ==============================================================
     all-rights-reserved  All Rights Reserved. This is not an open source license.
```

Comment:
Would it be possible to convert this document to .md. We have this issue with the stated goal to remove .rst and replace with .md files. A common approach to this is to convert a file when you touch it https://github.com/mozilla/addons/issues/9389

Not a requirement but could be an additional nice to have.

Reply:
Let's do this in a separate commit.